### PR TITLE
making FAQ section collapsible

### PR DESCRIPTION
**Pull request by ashishsingh18**
making FAQ section collapsible

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# Frequently Asked Questions (FAQ)
+**Frequently Asked Questions (FAQ)**
```
Comment (sarthakpati) [inline FAQ.md]: This change makes the text look bold, instead of a separate section (which it was before). Could you please revert this change?

Comment (ashishsingh18) [reply FAQ.md]: This will make that line look too big and bold too.

Comment (ashishsingh18) [reply FAQ.md]: As before, it's a separate section now.
